README.md

```diff
@@ -33,6 +33,12 @@ If the server starts up successfully you will see log messages like this:
 2021-06-30T05:59:56-07:00 [Web server] Listening at http://localhost:8080
 ```
 
+### Setting up HomeAssistant Addon (only for non-native docker installs)
+
+[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fneilenns%2Fambientweather2mqtt)
+
+[HA Add-On Installation](hassio_aw2m/README.md)
```

Comment:
There's only two bullet points in the secondary readme, can it just be included directly here instead of as a separate file?